src/components/CurriculumComponents/CurriculumDownloadView/SignedInFlow.tsx

```diff
@@ -82,6 +89,11 @@ export default function SignedInFlow({
           <Terms />
         </OakBox>
       </OakBox>
+      {submitError && (
+        <OakFieldError>
```

Comment:
Can I be a bit cheeky and ask for the submit button to be associated with this error message (when present) using `aria-describedby`? Feel free to say no and make me put it in a proper ticket, I don't mind.